<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderCreator</class>
 <widget class="QMainWindow" name="FolderCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Folder Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="projectTypeGroup">
      <property name="title">
       <string>Project Type</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radio3D">
         <property name="text">
          <string>3D Printing/Design</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioProgramming">
         <property name="text">
          <string>Programming</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page3D">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="group3D">
          <property name="title">
           <string>3D Printing Folders</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkSTL">
             <property name="text">
              <string>STL Files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkDesigns">
             <property name="text">
              <string>Designs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkGcode">
             <property name="text">
              <string>G-code</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkSources">
             <property name="text">
              <string>Sources</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkPrints">
             <property name="text">
              <string>Print Logs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkSettings">
             <property name="text">
              <string>Printer Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageProgramming">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <widget class="QWidget" name="tabGeneral">
           <attribute name="title">
            <string>General</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="checkSrc">
              <property name="text">
               <string>src (Source code files)</string>
              </property>
             </widget>
            </item>
             <item>
             <widget class="QCheckBox" name="checkLib">
              <property name="text">
               <string>lib (External libraries)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkDocs">
              <property name="text">
               <string>docs (Documentation)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkTests">
              <property name="text">
               <string>tests (Test scripts)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkConfig">
              <property name="text">
               <string>config (Configuration files)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkAssets">
              <property name="text">
               <string>assets (Media files)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabWeb">
           <attribute name="title">
            <string>Web</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkStatic">
              <property name="text">
               <string>static (Static files)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkViews">
              <property name="text">
               <string>views (Templates)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkComponents">
              <property name="text">
               <string>components (Reusable components)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabML">
           <attribute name="title">
            <string>ML/Data</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkNotebooks">
              <property name="text">
               <string>notebooks (Jupyter notebooks)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkModels">
              <property name="text">
               <string>models (Trained models)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkDatasets">
              <property name="text">
               <string>datasets (Raw/processed data)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSelectPath">
        <property name="text">
         <string>Select Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreate">
      <property name="text">
       <string>Create Folders</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
